fdbserver/MasterInterface.h

```diff
@@ -180,19 +180,18 @@ struct GetCommitVersionRequest {
 
 struct ReportRawCommittedVersionRequest {
 	constexpr static FileIdentifier file_identifier = 1853148;
-	SpanID spanContext;
 	Version version;
 	bool locked;
 	Optional<Value> metadataVersion;
 
 	ReplyPromise<Void> reply;
 
-	ReportRawCommittedVersionRequest() : spanContext(), version(invalidVersion), locked(false) {}
-	ReportRawCommittedVersionRequest(SpanID spanContext, Version version, bool locked, Optional<Value> metadataVersion) : spanContext(spanContext), version(version), locked(locked), metadataVersion(metadataVersion) {}
+	ReportRawCommittedVersionRequest() : version(invalidVersion), locked(false) {}
```

Comment:
a descriptive commit message will be appreciated. :)

Reply:
Done.